### DISCUSSION TITLE
如何在围栏式代码块中插入HTML代码？

### DISCUSSION THREAD

**Discussion by Gaobang100** [vuepress-theme-hope/vuepress-theme-hope#2633] [Q&A] [answered]
我需要在一个围栏式代码块中插入上下标。效果大概像这样：

\`\`\`
一些文本\<sub\>下标\</sub\>
\`\`\`

我应该如何实现呢？
如果没有这样的功能，请您告诉我应该如何用纯HTML实现？（我试了好久都不行呜呜呜）

**Comment by Mister-Hope** [comment, marked as answer]
You'd better understand "What's Code block" and what's it for?

**Reply by Gaobang100** [reply]
我表述不清楚。。
___
因为这样的渲染会让内容更突出，而且包含复制按钮，所以像这样的效果
![image](https://user-images.githubusercontent.com/105471582/210902506-a339f5ec-ca1d-426a-a6a9-4ae53bc85c6c.png)
会被渲染成怎样的HTML？我可以直接使用这段HTML吗？
这样应该可以插入上下标了

**Reply by Mister-Hope** [reply]
同样的回答，先了解一下代码块是什么，它的设计目标是什么。

**Reply by Gaobang100** [reply]
谢谢，现在我已经了解，并且明白了为什么这个问题无法实现。
___
不过刚才试了一下，只需要复制HTML，就可以实现这样的效果
![image](https://user-images.githubusercontent.com/105471582/210937743-52d08e0a-c7e8-46fa-8f8c-06b2207ddf4d.png)

另外问个小问题：我觉得 **普通代码块** 和 **引用** 的渲染比较暗，而这些内容恰好是我想突出的，我可以在哪里修改渲染样式？  
C++没有高亮？（其他语言正常）我应该怎样做？
![image](https://user-images.githubusercontent.com/105471582/210938560-fa280989-010f-4fcb-8d5f-2534d615f6e2.png)

**Reply by Mister-Hope** [reply]
C++的语言扩展是cpp，加号不是合法的文件字符

**Reply by Mister-Hope** [reply]
这种 noob 问题建议自己百度。

另外主题文档写了如何自定义样式，并支持你配置代码块主题。
